src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+}
+
+main {
+  border: 2px black solid;
+  margin: 0 auto;
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: space-between;
+  align-items: center;
+}
+
+main #clear-all {
+  border: 2px white solid;
+  height: 50px;
+  width: 100%;
+}
```

Comment:
- [ ] [REQUIRED] You have really done a great job implementing the design, however, it is required to style this project according to the wireframe provided. Check the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) for a picture of the wireframe.